client/client_pool/src/concord_client_pool.cpp

```diff
@@ -348,7 +348,7 @@ void ConcordClientPool::CreatePool(concord::config_pool::ConcordClientPoolConfig
   jobs_queue_max_size_ = config.external_requests_queue_size;
 }
 
-void ConcordClientPool::OnBatchingTimeout(const ClientPtr &client) {
+void ConcordClientPool::OnBatchingTimeout(std::shared_ptr<concord::external_client::ConcordClient> client) {
```

Comment:
-> ClientPtr

Reply:
It does not pass the clang-tidy check when changing to ClientPtr.

Reply:
I see